astroquery/vo_conesearch/conesearch.py

```diff
@@ -1,10 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-"""
-Support VO Simple Cone Search capabilities.
-
-.. note:: This maintains a similar API as ``astropy.vo.client``.
-
-"""
+"""Support VO Simple Cone Search capabilities."""
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 import six
```

Comment:
Clean up time if you want. But actually it's reasonable to leave that as future fun :)

Reply:
Done! 😄